### PR TITLE
Splitup comment see and manage permissions for motion comments

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/motions/apps.py [hunk 1]
```diff
@@ -14,19 +14,40 @@ class MotionsAppConfig(AppConfig):
...
         config.update_config_variables(get_config_variables())
         register_projector_elements(get_projector_elements())
 
         # Connect signals.
-        post_migrate.connect(create_builtin_workflows, dispatch_uid='motion_create_builtin_workflows')
+        post_migrate.connect(
+            create_builtin_workflows,
+            dispatch_uid='motion_create_builtin_workflows')
+        post_permission_creation.connect(
+            delete_old_comment_permission,
```
Comment (normanjaeckel) [inline openslides/motions/apps.py]: This function has nothing to do with appstart but with migration process. So I prefer not to pollute the code here with this stuff ...

File: openslides/motions/signals.py [hunk 2]
```diff
@@ -133,3 +134,10 @@ def required_users(sender, request_user, **kwargs):
             submitters_supporters.update(full_data['submitters_id'])
             submitters_supporters.update(full_data['supporters_id'])
     return submitters_supporters
+
+
+def delete_old_comment_permission(sender, **kwargs):
```
Comment (normanjaeckel) [inline openslides/motions/signals.py]: See above. Please do not run this stuff here.

File: openslides/motions/views.py [hunk 3]
```diff
@@ -138,7 +138,7 @@ def create(self, request, *args, **kwargs):
                     del request.data[key]
 
         # Check permission to send comment data.
-        if not has_perm(request.user, 'motions.can_see_and_manage_comments'):
+        if not has_perm(request.user, 'motions.can_manage_comments'):
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: The user should have to have both perms.

File: openslides/motions/views.py [hunk 4]
```diff
@@ -178,7 +178,7 @@ def update(self, request, *args, **kwargs):
         # Check permissions.
         if (not has_perm(request.user, 'motions.can_manage') and
                 not (motion.is_submitter(request.user) and motion.state.allow_submitter_edit) and
-                not has_perm(request.user, 'motions.can_see_and_manage_comments')):
+                not has_perm(request.user, 'motions.can_manage_comments')):
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Here the same.

File: openslides/motions/views.py [hunk 5]
```diff
@@ -199,7 +199,7 @@ def update(self, request, *args, **kwargs):
             for key in keys:
                 if key not in whitelist:
                     del request.data[key]
-        if not has_perm(request.user, 'motions.can_see_and_manage_comments'):
+        if not has_perm(request.user, 'motions.can_manage_comments'):
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Here the same.

**Comment by normanjaeckel** [comment]
Also CHANGELOG entry might be necessary.

File: openslides/motions/apps.py (hunk 1 above)
Comment (FinnStutzenstein) [reply openslides/motions/apps.py]: Done.

File: openslides/motions/views.py (hunk 3 above)
Comment (FinnStutzenstein) [reply openslides/motions/views.py]: Done.

File: openslides/motions/views.py (hunk 4 above)
Comment (FinnStutzenstein) [reply openslides/motions/views.py]: Done.

File: openslides/motions/views.py (hunk 5 above)
Comment (FinnStutzenstein) [reply openslides/motions/views.py]: Done.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Please review again.
The second commit can be ignored, because these changes are redone in the third commit. (It does not work, need another approach)

File: openslides/motions/migrations/0005_auto_20180202_1318.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+# Generated by Django 1.10.8 on 2018-02-02 12:18
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Permission
+from django.db import migrations
+
+
+def delete_old_comment_permission(apps, schema_editor):
+    """
+    Deletes the old 'can_see_and_manage_comments' permission which is
```
Comment (normanjaeckel) [inline openslides/motions/migrations/0005_auto_20180202_1318.py]: Ok so, but now nobody gets the new permission, so former admins are not able to handle comments any more.

File: openslides/motions/views.py [hunk 7]
```diff
@@ -178,7 +179,8 @@ def update(self, request, *args, **kwargs):
         # Check permissions.
         if (not has_perm(request.user, 'motions.can_manage') and
                 not (motion.is_submitter(request.user) and motion.state.allow_submitter_edit) and
-                not has_perm(request.user, 'motions.can_see_and_manage_comments')):
+                not has_perm(request.user, 'motions.can_see_comments') and
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Bool combination is wrong, should be (not (has_perm x and has_perm y)).

Comment (FinnStutzenstein) [reply openslides/motions/views.py]: Done.

File: openslides/motions/migrations/0005_auto_20180202_1318.py (hunk 6 above)
Comment (FinnStutzenstein) [reply openslides/motions/migrations/0005_auto_20180202_1318.py]: Done.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel I've reduced the test coverage limit in fact that we do not test migrations and this migration added some lines of code, so coverage fails.

File: openslides/motions/migrations/0005_auto_20180202_1318.py [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+            codename='can_see_comments',
+            name='Can see comments',
+            content_type=content_type)
+        perm_manage = Permission.objects.create(
+            codename='can_manage_comments',
+            name='Can manage comments',
+            content_type=content_type)
+
+        for group in groups:
+            print(group.name)
```
Comment (normanjaeckel) [inline openslides/motions/migrations/0005_auto_20180202_1318.py]: Please remove this print.

**Comment by normanjaeckel** [comment]
One remark. Then rebase and merge.

File: openslides/motions/migrations/0005_auto_20180202_1318.py (hunk 8 above)
Comment (FinnStutzenstein) [reply openslides/motions/migrations/0005_auto_20180202_1318.py]: Done.

**Comment by emanuelschuetze** [comment]
Successfully retested - with migration.

**Pull request merged**